python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -292,6 +308,19 @@ def make_time_major(tensor, drop_last=False):
             max_seq_len=self.config["model"]["max_seq_len"],
             batch_divisibility_req=self.config["sample_batch_size"])
 
+        pi_loss_abs = tf.abs(self.loss.pi_loss)
+        vf_loss_abs = tf.abs(self.loss.vf_loss * self.config["vf_loss_coeff"])
+        en_loss_abs = tf.abs(self.loss.entropy * self.config["entropy_coeff"])
+        lp_loss_abs = tf.abs(self.model.lp_loss)
```

Comment:
Models that don't have lp_loss won't work with this

Reply:
Fixed.